src/renderer/components/deposit/add/PendingAssetsWarning.stories.tsx

```diff
@@ -6,7 +6,7 @@ import * as FP from 'fp-ts/lib/function'
 
```

Comment:
Just moving existing content of `Deposit.subcomponents.stories` into `PendingAssetsWarning.stories.tsx` - changes can be ignored